### PR TITLE
Consistent spelling of neighboring to fix #865

### LINKED ISSUE #865: Inconsistent spelling: Neighbouring vs. Neighboring
>Thank you for your feedback! Please add below details so that we can help you a bit faster.

---

- [X] **I have reviewed and accepted the [guidelines for contributing](https://github.com/CellularPrivacy/Android-IMSI-Catcher-Detector/blob/development/.github/CONTRIBUTING.md) to this project.**
- [X] **I have searched the [open Issues](https://github.com/CellularPrivacy/Android-IMSI-Catcher-Detector/issues) and made sure I am not filing a duplicate.**
- [X] **I have read the notes on [debugging](https://github.com/CellularPrivacy/Android-IMSI-Catcher-Detector/blob/development/.github/CONTRIBUTING.md#debugging) and will properly maintain this Issue.**

---

While fixing #848, it came to my attention that our code and documentation has been invaded by a creepy spelling bug: We currently use both `neighbouring` *and* `neighboring` on our repository. This inevitably leads to coding and translation problems (therefore our `development` branch is currently broken). We need to decide on this ASAP so that everyone knows which spelling to use in the future.

* I also found this note in `CellTracker.java`:

```
[  ] Attention to the spelling of "neighbor" (USA) Vs. "neighbour" (Eng world)
```
---

This is how the Android Developer Resource is spelling it: `Neighboring`. Shall we go with that?

https://developer.android.com/reference/android/telephony/NeighboringCellInfo.html

---

@larsgrefer, you also use `neighbour` in your diagram. I would like to go with the most plausible solution here. Do you agree that we should be using `neighboring` everywhere from now on? If so, please send me an updated version of your diagram via E-Mail and I will fix our whole codebase today! Thank you.

### PR DESCRIPTION
**Pull request by SecUpwN**
>Thank you for submitting a pull request! Please add below details so that we can merge it faster.

---

- [X] **I have reviewed and accepted the [guidelines for contributing](https://github.com/CellularPrivacy/Android-IMSI-Catcher-Detector/blob/development/.github/CONTRIBUTING.md) to this project.**
- [X] **I have reviewed and closely followed the [Style Guide](https://github.com/CellularPrivacy/Android-IMSI-Catcher-Detector/wiki/Style-Guide) for this Android app.**

---

This pull request shall fix the inconsistent spelling of `neighboring` in code and documentation.

---

- [ ] Bugfix (non-breaking change which fixes an existing issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactor (restructuring of existing code without changing its external functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

---

#865

### CODE REVIEW

File: AIMSICD/src/main/res/values/translatable_strings.xml [hunk 1]
```diff
@@ -42,8 +42,8 @@
     <string name="device_info_title">Device Information</string>
     <string name="sim_info_title">SIM Information</string>
     <string name="network_info">Network Information</string>
-    <string name="cell_info_title">Neighboring Cells</string>
-    <string name="neighboring_cells_title">Neighboring Cells</string>
+    <string name="cell_info_title">Neighbouring Cells</string>
```
Comment (SecUpwN) [inline AIMSICD/src/main/res/values/translatable_strings.xml]: @larsgrefer, how come you're re-introducing neighb**ou**ring? I aimed for *one* unified word: `neighboring`.

File: AIMSICD/src/main/res/values/translatable_strings.xml [hunk 2]
```diff
@@ -318,7 +318,7 @@
 
 
     <!-- Section Main -->
-    <string name="help_main_neighboring_cells">Shows neighboring cells.</string>
+    <string name="help_main_neighboring_cells">Shows neighbouring cells.</string>
```
Comment (SecUpwN) [inline AIMSICD/src/main/res/values/translatable_strings.xml]: Please change that back as well. I don't see a reason why we should do this at all, we should stick with one word.

Comment (agilob) [inline AIMSICD/src/main/res/values/translatable_strings.xml]: @SecUpwN he added new translation file that will create distinction between UK English and American English... Now I wonder how weblate will handle it. Lets see

Comment (SecUpwN) [inline AIMSICD/src/main/res/values/translatable_strings.xml]: >distinction between UK English and American English

The whole point of this pull request is to unify this so taht we don't need to distinguish anything...

Comment (agilob) [inline AIMSICD/src/main/res/values/translatable_strings.xml]: My Android doesn't agree with you 
how-to-change-the-language-on-android-from-chinese-to-english-03


Why are there different English and German dialects supported by Android?

Comment (SecUpwN) [inline AIMSICD/src/main/res/values/translatable_strings.xml]: >Why are there different English and German dialects supported by Android?

Yes, Android may do that. But what I wanted to accomplish, was a unified version of this **one** word to avoid developers using other spelling and re-introducing new inconsistencies.

Comment (agilob) [inline AIMSICD/src/main/res/values/translatable_strings.xml]: You're vandalising English dictionary. Or maybe vandalizing?

Comment (SecUpwN) [inline AIMSICD/src/main/res/values/translatable_strings.xml]: >You're vandalising English dictionary. Or maybe vandalizing? :>

Ha, our whole project can be considered "vandalism" on the minds of spies.
And not to forget Google themselves, since they still refuse to add a [ciphering indicator](https://code.google.com/p/android/issues/detail?id=5353).

Comment (agilob) [inline AIMSICD/src/main/res/values/translatable_strings.xml]: There are at least 3 ways how you can get ciphering indicator in Android. Two are supported by AIMSICD and buggy, so it doesn't work. Still, change made by @larsgrefer  is good.

File: AIMSICD/src/main/res/values-en-rUS/translatable_strings.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="cell_info_title">Neighboring Cells</string>
```
Comment (larsgrefer) [inline AIMSICD/src/main/res/values-en-rUS/translatable_strings.xml]: @SecUpwN These strings here will be used in an en-US locale (aka american english)

Comment (SecUpwN) [inline AIMSICD/src/main/res/values-en-rUS/translatable_strings.xml]: @larsgrefer, thanks for clarifying, makes sense. But do you think why did you change [this](https://github.com/CellularPrivacy/Android-IMSI-Catcher-Detector/blob/NeighboringConsistency/AIMSICD/src/main/res/values/translatable_strings.xml#L321) again?

Comment (larsgrefer) [inline AIMSICD/src/main/res/values-en-rUS/translatable_strings.xml]: @SecUpwN so all non-american english devices get the non-american variant with *ou*

Comment (SecUpwN) [inline AIMSICD/src/main/res/values-en-rUS/translatable_strings.xml]: >so all non-american english devices get the non-american variant with ou

Is it that big of a deal if we only use `neighboring` to unify this once and for all? Furthermore, did it even make sense renaming and replacing the string naames everywhere then? Don't get me wrong, I know the reasoning you have behind this, I just don't want developers to get confused...

Comment (SecUpwN) [inline AIMSICD/src/main/res/values-en-rUS/translatable_strings.xml]: >so all non-american english devices get the non-american variant with ou

Oh, now I get it! Let's see how Weblate reacts to the changes. Approved!

**Comment by SecUpwN** [comment]
>so all non-american english devices get the non-american variant with ou

Oh, now I get it! Let's see how Weblate reacts to the changes. Approved!

**Comment by SecUpwN** [comment]
Approved
